bin/extract.sh

```diff
@@ -10,6 +10,16 @@ do
     echo "...  $name"
     cp $corpus txt/$name.txt
 done
+sed -i '/\[Illustration\]/d'  txt/*.txt
+sed -i '1,/START/d' txt/*.txt
+sed -i '/END OF THE PROJECT GUTENBERG/,$d' txt/*.txt
+sed -i '/End of Project/d' txt/*.txt
+sed -i '/\*       \*/d' txt/*.txt 
+for f in txt/*-[1-9].txt
+do
+    tome=$(echo "$f" | sed 's/-[1-9].txt//g');
+    cat "$tome"-[1-9].txt > "$tome".txt
+done
```

Comment:
Nice loop! Two suggestions:
- I think `tome` is not meaningful in English, maybe `volume` would be a better fit?
- please delete the `*-[1-9].txt` files when the concatenation is done